iphone/Classes/TiUIiPhoneProxy.m

```diff
@@ -226,10 +226,12 @@ -(NSString*)apiName
 
 BEGIN_UI_THREAD_PROTECTED_VALUE(statusBarHidden,NSNumber)
 result = [NSNumber numberWithBool:[[UIApplication sharedApplication] isStatusBarHidden]];
+DEPRECATED_REPLACED_REMOVED(@"UI.iPhone.statusBarHidden",@"3.1.3", @"6.0.0", @"UI.Window.fullscreen");
```

Comment:
Ok, the macros around it generate the method, get it. The property should not even be on the iPhone namespace anymore, since its **removed** since 3.1.3 (http://docs.appcelerator.com/platform/latest/#!/api/Titanium.UI.iPhone-property-statusBarHidden). No one should use it anymore.